auth/postgres/init.go

```diff
@@ -4,44 +4,12 @@
 package postgres
 
 import (
-	"fmt"
-
-	_ "github.com/jackc/pgx/v5/stdlib" // required for SQL access
-	"github.com/jmoiron/sqlx"
 	migrate "github.com/rubenv/sql-migrate"
 )
 
-// Config defines the options that are used when connecting to a PostgreSQL instance
-type Config struct {
-	Host        string
-	Port        string
-	User        string
-	Pass        string
-	Name        string
-	SSLMode     string
-	SSLCert     string
-	SSLKey      string
-	SSLRootCert string
-}
-
-// Connect creates a connection to the PostgreSQL instance and applies any
-// unapplied database migrations. A non-nil error is returned to indicate failure.
-func Connect(cfg Config) (*sqlx.DB, error) {
-	url := fmt.Sprintf("host=%s port=%s user=%s dbname=%s password=%s sslmode=%s sslcert=%s sslkey=%s sslrootcert=%s", cfg.Host, cfg.Port, cfg.User, cfg.Name, cfg.Pass, cfg.SSLMode, cfg.SSLCert, cfg.SSLKey, cfg.SSLRootCert)
-
-	db, err := sqlx.Open("pgx", url)
-	if err != nil {
-		return nil, err
-	}
-
-	if err := migrateDB(db); err != nil {
-		return nil, err
-	}
-	return db, nil
-}
-
-func migrateDB(db *sqlx.DB) error {
-	migrations := &migrate.MemoryMigrationSource{
+// Migration of auth service.
```

Comment:
Inline import: `import migrate "github.com/rubenv/sql-migrate"`. Please check for other similar cases.